src/ui.rs

```diff
@@ -36,16 +41,18 @@ pub enum Action {
     Move,
     /// Delete indicates the app should delete the image in response to this event
     Delete,
+    /// Put number on Screen for Numeric operations
+    NumericOp(Token),
+    /// Copy multiple images
+    CopyMultiple(&'a Vec<String>),
```

Comment:
Not sure I like this design. I don't think `Copy` and `CopyMultiple` should be separate variants. I'd prefer copy took a number, and the default was 1. It shouldn't be its responsibility to do anything with a reference to a string vector.

I'm also not sure we need a string vector here. Logic could be: First time a number comes in, set Action to NumericOp with that number. Then there's two options, if a non number comes in, dispatch the action with the number we have, or if a number comes in, multiply the existing number by 10 and add the new one.